src/storage/btree.c

```diff
@@ -6498,6 +6498,7 @@ btree_get_stats_key (THREAD_ENTRY * thread_p, BTREE_STATS_ENV * env, MVCC_SNAPSH
   LEAF_REC leaf_pnt;
   bool clear_key = false;
   int offset;
+  int max_visible_oids, num_visible_oids;
```

Comment:
Nit. I would put the declaration in the scope it's used.

Reply:
I think that declaring the variables in one place is easy to read. Personally, it is convenient to see the declarations at a glance.